networkconfig/holesky-stage.go

```diff
@@ -11,7 +11,8 @@ import (
 var HoleskyStage = NetworkConfig{
 	Name:                 "holesky-stage",
 	Beacon:               beacon.NewNetwork(spectypes.HoleskyNetwork),
-	Domain:               [4]byte{0x00, 0x00, 0x31, 0x99}, // TODO: (Alan) revert
+	GenesisDomainType:    [4]byte{0x00, 0x00, 0x31, 0x12},
+	AlanDomainType:       [4]byte{0x00, 0x00, 0x31, 0x99},
```

Comment:
why not `0x13` (leave space for next forks)

Reply:
done